### PR TITLE
ovirt_vms: Add new cloud_init_nics parameter (#3447)

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

ovirt_vms

##### ANSIBLE VERSION

```
ansible 2.3.0
```

##### SUMMARY

#3447 

```

```

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
@mwperina @ryansb

File: cloud/ovirt/ovirt_vms.py [hunk 1]
```diff
@@ -191,6 +191,20 @@
             - "C(nic_gateway) - If boot protocol is static, set this gateway to network interface of Virtual Machine."
             - "C(nic_name) - Set name to network interface of Virtual Machine."
             - "C(nic_on_boot) - If I(True) network interface will be set to start on boot."
+    cloud_init_nics:
+        description:
+            - "List of dictionaries representing network interafaces to be setup by cloud init."
+            - "This option is used, when user need to setup more network interfaces via cloud init."
```
Comment (mwperina) [inline cloud/ovirt/ovirt_vms.py]: ... when user needs ...

File: cloud/ovirt/ovirt_vms.py [hunk 2]
```diff
@@ -191,6 +191,20 @@
...
+            - "If one network interface is enough, user should use C(cloud_init) I(nic_*) parameters. C(cloud_init) I(nic_*) parameters
+               are merged with C(cloud_init_nics) parameters."
+            - "Dictionary can contain following values:"
+            - "C(nic_boot_protocol) - Set boot protocol of the network interface of Virtual Machine. Can be one of None, DHCP or Static."
```
Comment (mwperina) [inline cloud/ovirt/ovirt_vms.py]: Maybe it's better to display value in lowercase as values are not case sensitive: none, dhcp, static

**Review by mwperina — CHANGES_REQUESTED** [review]

**Pull request merged**